analyzer/src/traversal/expressions.rs

```diff
@@ -556,7 +556,20 @@ fn expr_unary_operation(
                     Location::Value,
                 ))
             }
-            _ => todo!(),
+            UnaryOperator::Invert => {
+                context.not_yet_implemented("unary invert", exp.span);
+                Ok(ExpressionAttributes::new(
+                    Type::unit(),
+                    Location::Value,
+                ))
+            },
+            UnaryOperator::UAdd => {
```

Comment:
what is this supposed to do?

Reply:
as far as I can tell, this is sometimes used to convert things to numbers.

is there another use case that Im missing? if not, I think we should just ditch it.